pkg/modules/dependencies.go

```diff
@@ -6,10 +6,12 @@ const (
 
 	Core             string = "core"
 	GrafanaAPIServer string = "grafana-apiserver"
+	StorageAPIServer string = "storage-apiserver"
```

Comment:
```suggestion
	StorageService string = "storage-service"
```
or similar?  It is not an apiserver, so likely better we do not call it one.

`StorageAPI`, `StorageRPC`?